codespell_lib/data/dictionary.txt

```diff
@@ -12751,6 +12757,7 @@ initilize->initialize
 initilized->initialized
 initilizes->initializes
 initilizing->initializing
+inititalization->initialization
```

Comment:
Please can you add the BE variant with an `s`

Reply:
Done

Reply:
You've added the plural @Gelma , but please can we have the BE versions of both, i.e. `inititalisation->initialisation` and `inititalisations>initialisations`